Core/DefInjected/ThingDef/Apparel_Headgear.xml

```diff
@@ -51,9 +56,9 @@
   <!-- EN: An artistically terrifying war mask. It hides the wearer's face and makes them more confident. -->
   <Apparel_WarMask.description>Искусно украшенная и устрашающе выглядящая боевая маска. Скрывает лицо носящего и даёт чувство уверенности.</Apparel_WarMask.description>
   
-  <!-- EN: war veil -->
-  <Apparel_WarVeil.label>боевой покров</Apparel_WarVeil.label>
-  <!-- EN: A war veil, made out of reinforced cloth torn to appear damaged in combat. Some believe this functions as a talisman of courage. -->
-  <Apparel_WarVeil.description>Боевой покров, изготовленный из укреплённой ткани. Надорван, чтобы выглядеть повреждённым в бою. Некоторые верят, что он придаёт храбрости носящему.</Apparel_WarVeil.description>
+  <!-- EN: veil -->
+  <Apparel_WarVeil.label>покров</Apparel_WarVeil.label>
+  <!-- EN: A piece of fabric which covers the wearer's face. -->
+  <Apparel_WarVeil.description>Покров, изготовленный из укреплённой ткани. Надорван, чтобы выглядеть повреждённым в бою. Некоторые верят, что он придаёт храбрости носящему.</Apparel_WarVeil.description>
```

Comment:
```suggestion
  <Apparel_WarVeil.description>Покров, изготовленный из укреплённой ткани.</Apparel_WarVeil.description>
```

Reply:
A piece of fabric which covers the wearer's face
```suggestion
  <Apparel_WarVeil.description>Кусок ткани, закрывающий лицо носящего.</Apparel_WarVeil.description>
```